src/middleware/packages/ldp/services/resource/actions/delete.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
       await ctx.call('ldp.container.detach', { containerUri, resourceUri, webId: 'system' });
     }
 
-    if (oldData['@type'] === 'semapps:File') {
+    if (oldData['type'] === 'File') {
```

Comment:
Are you sure ? I don't think the type will ever be `'File'`, except if you use a JSON-LD context of your own.
Currently handling JSON-LD is a mess, which is why we are thinking of using a (custom ?) library for that.

Reply:
Yes, I updated. Only the @ was incorrect